acapy_agent/commands/tests/test_help.py

```diff
@@ -5,22 +5,20 @@
 
 class TestHelp(IsolatedAsyncioTestCase):
     def test_exec_help(self):
-        with mock.patch.object(
-            command.ArgumentParser, "print_help"
-        ) as mock_print_help, mock.patch(
-            "builtins.print", mock.MagicMock()
-        ) as mock_print:
+        with (
+            mock.patch.object(command.ArgumentParser, "print_help") as mock_print_help,
+            mock.patch("builtins.print", mock.MagicMock()) as mock_print,
+        ):
             command.execute([])
             mock_print_help.assert_called_once()
 
             command.execute(["-v"])
             mock_print.assert_called_once_with(command.__version__)
 
     def test_main(self):
-        with mock.patch.object(
-            command, "__name__", "__main__"
-        ) as mock_name, mock.patch.object(
-            command, "execute", mock.MagicMock()
-        ) as mock_execute:
+        with (
+            mock.patch.object(command, "__name__", "__main__") as mock_name,
```

Comment:
I've started removing these variables (`mock_name`) when they aren't used. It's not important but makes the code slightly cleaner. Doesn't have to happen in this PR.